### PR TITLE
feat(route/my-formosa): Create route

### PR DESCRIPTION
**Pull request by dzx-dzx**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/my-formosa
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/my-formosa/index.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['my-formosa.com/'],
+        },
+    ],
+    name: '美麗島電子報',
+    maintainers: [''],
```
Comment (TonyRL) [inline lib/routes/my-formosa/index.ts:27]: Missing `maintainers`

File: lib/routes/my-formosa/index.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['my-formosa.com/'],
+        },
+    ],
+    name: '美麗島電子報',
```
Comment (TonyRL) [inline lib/routes/my-formosa/index.ts:26]: `name` shouldn't be the same as namespace's name

File: lib/routes/my-formosa/index.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/',
+    categories: ['traditional-media'],
```
Comment (TonyRL) [inline lib/routes/my-formosa/index.ts:10]: It belongs to `new-media`

File: lib/routes/my-formosa/index.ts [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    example: '/my-formosa/',
```
Comment (TonyRL) [inline lib/routes/my-formosa/index.ts:11]: No trailing slash

File: lib/routes/my-formosa/index.ts [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                const res = await fetch(link);
+                const $ = load(res);
+
+                return {
+                    title,
+                    link,
+                    author: $('.page-header~#featured-news h4').text(),
+                    category: $("meta[name='keywords']").attr('content').split(','),
+                    pubDate: timezone(parseDate($('.date').text()), +8),
+                    description: $('.body').html(),
```
Comment (TonyRL) [inline lib/routes/my-formosa/index.ts:64]: Missing description for `http://www.my-formosa.com/TV_*.htm`

File: lib/routes/my-formosa/index.ts [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+                const res = await fetch(link);
+                const $ = load(res);
+
+                return {
+                    title,
+                    link,
+                    author: $('.page-header~#featured-news h4').text(),
+                    category: $("meta[name='keywords']").attr('content').split(','),
+                    pubDate: timezone(parseDate($('.date').text()), +8),
+                    description: $('.body').html() ?? $('video-post').html(),
```
Comment (TonyRL) [inline lib/routes/my-formosa/index.ts:64]: - Empty pubDate
- Still missing description for `http://www.my-formosa.com/TV_*.htm`

File: lib/routes/my-formosa/index.ts [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+                const $ = load(res);
+
+                const isTV = /^\/TV/.test(new URL(link).pathname);
+
+                return {
+                    title,
+                    link,
+                    author: $('.page-header~#featured-news h4').text(),
+                    category: $("meta[name='keywords']").attr('content').split(','),
+                    pubDate: timezone(parseDate(isTV ? $('.icon-calendar')[0].next.data.trim() : $('.date').text()), +8),
```
Comment (TonyRL) [inline lib/routes/my-formosa/index.ts:65]: pubDate is still [empty](https://github.com/DIYgod/RSSHub/pull/15838#issuecomment-2155307982).

Comment (dzx-dzx) [reply lib/routes/my-formosa/index.ts:65]: I don't understand... It works on my machine.

File: lib/routes/my-formosa/index.ts [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+                    const res = await fetch(link);
+                    const $ = load(res);
+
+                    const isTV = /^\/TV/.test(new URL(link).pathname);
+
+                    return {
+                        title,
+                        link,
+                        author: $('.page-header~#featured-news h4').text(),
+                        category: $("meta[name='keywords']").attr('content').split(','),
```
Comment (TonyRL) [inline lib/routes/my-formosa/index.ts:65]:
```suggestion
                        category: $("meta[name='keywords']").attr('content').split(',').filter(Boolean),
```

**Pull request merged**